core/domain/role_services.py

```diff
@@ -74,6 +77,9 @@
 ACTION_MANAGE_MEMCACHE = 'MANAGE_MEMCACHE'
 ACTION_MANAGE_QUESTION_RIGHTS = 'MANAGE_QUESTION_RIGHTS'
 ACTION_MANAGE_TOPIC_RIGHTS = 'MANAGE_TOPIC_RIGHTS'
+ACTION_MANAGE_BLOG_POST_EDITORS = 'MANAGE_BLOG_POST_EDITORS'
+ACTION_MODIFY_ROLES_FOR_ANY_ACTIVITY = 'MODIFY_ROLES_FOR_ANY_ACTIVITY'
```

Comment:
Why are these two added? Are they duplicates of the ones below?

Reply:
Came up due to merge conflict. Both of these were deleted in a recent PR. Removed.